src/shared/components/common/content-actions/content-action-dropdown.tsx

```diff
@@ -159,6 +159,26 @@ export default class ContentActionDropdown extends Component<
       (amMod(community.id) || (amAdmin() && community.local)) &&
       !creator_banned_from_community;
 
+    const modHistoryUserTranslation = I18NextService.i18n.t(
+      "user_moderation_history",
+      { user: creator.name },
+    );
+
+    // The link and translation string for the item
+    const { modHistoryItemLink, modHistoryItemTranslation } =
+      type === "post"
+        ? {
+            modHistoryItemLink: `/modlog?postId=${id}`,
+            modHistoryItemTranslation: I18NextService.i18n.t(
+              "post_moderation_history",
+            ),
+          }
+        : {
+            modHistoryItemLink: `/modlog?commentId=${id}`,
+            modHistoryItemTranslation: I18NextService.i18n.t(
+              "comment_moderation_history",
+            ),
+          };
```

Comment:
Are moderation histories for specific posts and comments something admins use? I can't imagine that any given post or comment will be removed and restored so many times that this would be useful.

Reply:
hrm... Ya I think it could be useful to see things that have been removed / restored multiple times. But more importantly, there might be more recent mod actions done on a person, and you might only want to see the mod history for that item, rather than going to the person and having to scroll through. Like a year old comment or something.